tests/unit/test_num.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Redis Ltd. 2018 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
+#include "src/util/num.h"
+
+#include "acutest.h"
+
+#define TYPE(width) uint##width ## _t
```

Comment:
How are spaces are treated here?
`uint##width` has no spaces while `width ## _t` does have space characters between
Are we expecting `TYPE(8)` to "return" `uint8_t` ?